libraries/matrixmedia/impl/src/main/kotlin/io/element/android/libraries/matrix/ui/media/ImageLoaderFactories.kt

```diff
@@ -16,32 +16,40 @@ import coil3.gif.GifDecoder
 import coil3.network.okhttp.OkHttpNetworkFetcherFactory
 import dev.zacsweers.metro.AppScope
 import dev.zacsweers.metro.ContributesBinding
-import dev.zacsweers.metro.Inject
 import dev.zacsweers.metro.Provider
 import io.element.android.libraries.di.annotations.ApplicationContext
 import io.element.android.libraries.matrix.api.media.MatrixMediaLoader
 import okhttp3.OkHttpClient
 
-interface LoggedInImageLoaderFactory {
+interface ImageLoaderFactory {
```

Comment:
Maybe we should `Builder` pattern here instead of having 2 methods?

Reply:
Maybe. I'll look at it in a follow up PR.